test/control-plane/services/dual-stack/generate.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Generate the golden test files for the DualStack test
+#
+
+set -eux
+
+export KUBECONFIG=kubeconfig
+
+manifests=(
+	# Creates the echo deployment with two echo replicas.
+	manifests/echo-dpl.yaml
+
+	# Allows all to echo.
+	manifests/echo-policy.yaml
+
+	# Creates dual-stack service for echo.
+	manifests/echo_svc_dualstack.yaml
+)
+
+: Start a kind cluster
```

Comment:
Why are these not just comments with `#`?

Reply:
This is a pattern used with `set -x` to show "comments" on what's happening.

Reply:
```
/tmp $ cat test.sh
#!/usr/bin/env bash
set -x

echo "hello"
: this is a : comment
# this is a # comment

/tmp $ ./test.sh
+ echo hello
hello
+ : this is a : comment
```

Reply:
:+1: